docs/server/features/archiving_blog_post.md

```diff
@@ -0,0 +1,248 @@
+# Archiving: Hands on!
```

Comment:
I think somewhere we should mention that archiving is simple to configure and use.  There's no specialness about archived data from the client point of view - it reads like any other data, just a bit slower due to the latency of the underlying object store.  This isn't just a cost play - it's also a scalability play and is really the foundation of separating storage from compute for KurrentDB.

Reply:
thanks! i've added these elements to the blog post